bazel/toolchain/Dockerfile.llvm

```diff
@@ -14,7 +14,7 @@ ADD --chmod=755 https://github.com/kadwanev/retry/raw/20997c7712a4/retry /usr/bi
 
 WORKDIR /opt
 
-ARG LLVM_VERSION=19.1.6
+ARG LLVM_VERSION=19.1.7
```

Comment:
so is this being overridden, or its irrelevant because are using pre-built v18?

Reply:
I've been overriding it when I build because I'm testing many different versions